pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -334,9 +334,9 @@ func (ctrl *Controller) generateOriginalKubeletConfig(role string) (*ignv2_2type
 
 func (ctrl *Controller) syncStatusOnly(cfg *mcfgv1.KubeletConfig, err error, args ...interface{}) error {
 	return retry.RetryOnConflict(updateBackoff, func() error {
-		newcfg, err := ctrl.mckLister.Get(cfg.Name)
-		if err != nil {
-			return err
+		newcfg, getErr := ctrl.mckLister.Get(cfg.Name)
+		if getErr != nil {
+			return getErr
 		}
 		newcfg.Status.Conditions = append(newcfg.Status.Conditions, wrapErrorWithCondition(err, args...))
```

Comment:
and maybe this needs to be wrapped in a `if err != nil` to avoid reporting nothing?

Reply:
wrapErrorWithCondition returns Success if error = nil (https://github.com/openshift/machine-config-operator/blob/master/pkg/controller/kubelet-config/helpers.go#L119).
So to me it looks like it'll always append a new condition here and we don't need to avoid this step,

Reply:
ah but looks like the return below ends up making the syncHandler return nil on no-error runs. Maybe we want to report an success/no errors reported msg here?